gulpfile.js

```diff
@@ -737,6 +737,7 @@ function release_deb(arch, appDirectory, done) {
                 `chown root:root ${LINUX_INSTALL_DIR}`,
                 `chown -R root:root ${LINUX_INSTALL_DIR}/${metadata.name}`,
                 `xdg-desktop-menu install ${LINUX_INSTALL_DIR}/${metadata.name}/${metadata.name}.desktop`,
+                `chmod -R +xr /opt/betaflight/betaflight-configurator/`,
```

Comment:
This `chmod -R +xr` will make all files in the directory executable, whereas we only want the two binaries, `/opt/betaflight/betaflight-configurator/chrome_crashpad_handler` and `/opt/betaflight/betaflight-configurator/betaflight-configurator` to be executable - not bitmaps, libraries etc.
So the three commands are needed, then the capital X in `chmod -R +Xr /opt/betaflight/betaflight-configurator/` sets the *directory only* to be executable which means that the user can read which files are inside the directory.